packages/components/src/internal/components/samples/actions.ts

```diff
@@ -140,15 +140,20 @@ export function deleteSampleSet(rowId: number): Promise<any> {
  * @param schemaQuery SchemaQuery which sources the request for rows
  * @param sampleColumn A QueryColumn used to map fieldKey, displayColumn, and keyColumn data
  * @param filterArray A collection of filters used when requesting rows
+ * @param displayValueKey Column name containing grid display value of Sample Type
+ * @param valueKey Column name containing grid value of Sample Type
  */
 export function fetchSamples(
     schemaQuery: SchemaQuery,
     sampleColumn: QueryColumn,
-    filterArray: Filter.IFilter[]
+    filterArray: Filter.IFilter[],
+    displayValueKey: string,
+    valueKey: string,
 ): Promise<OrderedMap<any, any>> {
     return selectRows({
         schemaName: schemaQuery.schemaName,
         queryName: schemaQuery.queryName,
+        columns: ['name', 'sampleId', 'RowId'],
```

Comment:
As discussed, this should request the `displayValueKey` and `valueKey` explicitly so they don't get "dropped" from the query.

```ts
columns: ['RowId', displayValueKey, valueKey],
```